data/events/scripts/party.lua

```diff
@@ -9,3 +9,24 @@ end
 function Party:onDisband()
 	return true
 end
+
+function Party:onShareExperience(exp)
+	local sharedExperienceMultiplier = 1.20 --20%
+	local vocationsIds = {}
+	table.insert(vocationsIds, self:getLeader():getVocation():getId())
```

Comment:
There should be a if statement around this (it shouldn't insert a vocationId if it's equal to VOCATION_NONE), and insert base vocationId instead (same goes for members).